### PR TITLE
Breadcrumb refactoring

### PR DESCRIPTION
**Pull request by shhsu**

### CODE REVIEW

File: src/app/common/feature-vtabs.component.ts [hunk 1]
```diff
@@ -57,7 +55,7 @@ export class GlobalModuleReference {
     // NOTE: when [routerLink] is used, Angular automatically set tabindex="0". In order to avoid that behavior, we decided to add tabindex="-1".
     template: `
 <div class="sidebar crumb" [class.nav]="IsActive">
```
Comment (jhkimnew) [inline src/app/common/feature-vtabs.component.ts]: please rename this to isActive from IsActive, too.

File: src/app/header/titles.component.ts [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
 
 @Component({
     selector: `titles`,
     template: `
 <div class="titles">
     <ul class="breadcrumbs">
         <li *ngFor="let crumb of breadcrumbs; index as i">
-            <span [ngClass]="{root: !i}" *ngIf="!(crumb.routerLink)">{{crumb.label}}</span>
-            <a *ngIf="crumb.routerLink" [routerLink]="crumb.routerLink">{{crumb.label}}</a>
+            <span [ngClass]="{root: !i}" [class.color-active]="canNavigate(crumb)" (click)="navigate(crumb)">{{crumb.label}}</span>
```
Comment (jhkimnew) [inline src/app/header/titles.component.ts]: I think you should add 'tabindex="0"' to make this focusable. Otherwise users can't navigate with keyboard.

Comment (jhkimnew) [reply src/app/header/titles.component.ts]: And add accessibility-focusable class as well to set the style when it is focused.

File: src/app/common/feature-vtabs.component.ts (hunk 1 above)
Comment (shhsu) [reply src/app/common/feature-vtabs.component.ts]: updated

**Review by jhkimnew — APPROVED** [review]

**Pull request merged**